test/e2e-go/features/incentives/suspension_test.go

```diff
@@ -135,10 +135,12 @@ func TestBasicSuspension(t *testing.T) {
 	lg, err := fixture.StartNode(c20.DataDir())
 	a.NoError(err)
 
-	// Wait for newly restarted node to start. Presumably it'll catchup in
-	// seconds, and propose by round 90
+	// Wait for newly restarted node to start.
 	stat, err := lg.Status()
 	a.NoError(err)
+	// Waiting for this round should show it has started and caught up.
+	stat, err = lg.WaitForRound(afterStop.LastRound + 55)
```

Comment:
where does the 55 come from?

Reply:
n20 has 20% of stake. 55 rounds is more than 50, which is how long it takes for a node with 20% of stake to be suspended (supposed to propose 1/5, so suspended if not 1/50).

`WaitForRound` uses absolute numbering, so we base everything off the round in which we stopped n20.